lib/public/views/RunTypes/RunTypes.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import RunTypeService from './RunTypesService.js';
+import { Observable } from '/js/src/index.js';
+
+/**
+ * General run types model
+ */
+export default class RunTypes extends Observable {
```

Comment:
Name it RunTypeModel (singular), to avoid conflicts with the backend sequelize model

Reply:
You have to update 
Model.js l.98
runTypeFilter l.20